apps/web/src/server/views/applications/case-documentation/folder/documentation-folder.njk

```diff
@@ -36,56 +40,16 @@
 					items: breadcrumbItems
 				}) }}
 
-				<div class="pins-dashboard-list--hide-controls">
-					<h2 class="govuk-heading-l">{{ currentFolder.displayNameEn }} folder</h2>
-
-					{% set folderRows = [] %}
-					{% for subFolder in subFolders %}
-						{% set folderRows = folderRows | concat ([[
-							{ html: '<a class="govuk-link govuk-body-m" href="' + 'document-category' |url({caseId: caseId, documentationCategory: subFolder}) +'">' + subFolder.displayNameEn + '</a>' }
-						]]) %}
-					{% endfor %}
-
-					{{ govukAccordion({
-						items: [{
-							content: {
-								html: govukTable({
-									rows: folderRows
-								})
-							}
-						}]
-					}) }}
-				</div>
-
-				{% set userIsInspector = domainType === 'inspector' %}
+				{{ folderHeader(currentFolder, subFolders, caseId) }}
 
-<div class="govuk-width-container">
```

Comment:
as I said in the previous comment: the S51 page can be complicated, because it looks like a "documentation-folder" but it's not.

For this task I was able to completely re-use the nunjucks, just adding a few IFs (i.e. if it is S51 show the button for the s51 creation; if it documentation show the button for the Upload).

The part that comes after (actions and table with the items) is going to be more complicated to re-use it and maybe we're going to need to create a whole new template.
For now, this should be fine.

Reply:
yeah I agree - you can probably abstract from the original the page few component and then create two distinct pages: s51 and documentation pages; and import the common components as it's needed on each